public/js/pimcore/object/tags/reverseObjectRelation.js

```diff
@@ -178,7 +178,7 @@ pimcore.object.tags.reverseObjectRelation = Class.create(pimcore.object.tags.man
                 markDirty: false,
                 listeners: {
                     afterrender: function (gridview) {
-                        this.requestNicePathData(this.store.data);
+                        this.requestNicePathData(this.store.data, true);
```

Comment:
After testing this, it worked loading the nice path also in getLayoutEdit (reload admin mode) and I did not encounter any disadvantages.

Reply:
:+1:  now it looks good. thx very much!